HousingFinanceInterimApi.Tests/HousingFinanceInterimApi.Tests.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="AutoFixture" Version="4.17.0" />
+    <PackageReference Include="dotenv.net" Version="3.1.2" />
```

Comment:
I don't think the tests project needs this - it has been getting about without it just fine.

Reply:
But this is the tests project. Any env vars that are needed have to be part of the given test's arrange step, or some general setup

Reply:
Oh right, good catch https://github.com/LBHackney-IT/housing-finance-interim-api/pull/136/commits/b09ed64b17c05b02ba097be9d2123455e28b81d1